api/services/administration/general/responses/createAWSAccountExternalId.yaml

```diff
@@ -64,7 +64,8 @@ content:
                       description: >
                         Array of data objects
                       items:
-                        - $ref: "../schemas/createAwsAccountExternalId-example1.yaml"
+                        allOf:
```

Comment:
AFAIK, `allOf` is needed only when you need to validate the object against multiple schemas. Since there's only once, wouldn't this work?

```
items:
    $ref: "../schemas/createAwsAccountExternalId-example1.yaml"
```

If this works let's prefer it over wrapping in `allOf`. Also applies to all other instances of this change.

Reply:
What you suggested does work in terms of presenting the example and it doesn't cause a semantic error. However, it changes the way the response looks in the documentation, see screenshots for the differences. I assume we don't want this outcome but let me know if you think otherwise
![original](https://user-images.githubusercontent.com/98812210/182015592-1ddd17a1-1c0f-49e4-a313-b34277b9dd99.png)
![updated](https://user-images.githubusercontent.com/98812210/182015595-9b3d4f31-c89e-48a2-bf7b-e0623649c4b2.png)
.